internal/logging/logging.go

```diff
@@ -23,16 +30,44 @@ func initLogger() {
 		logLevel = logrus.InfoLevel
 	}
 
-	log.Level = logLevel
-	log.Out = os.Stdout
-	log.ReportCaller = true
+	if cfg.LogFormater == "text" {
+		logger.Formatter = &logrus.TextFormatter{}
+	} else {
+		logger.Formatter = &logrus.JSONFormatter{}
+	}
+
+	logger.Level = logLevel
+	logger.Out = os.Stdout
+	logger.ReportCaller = true
+
+	if cfg.CwAccessKey != "" {
+		cred := credentials.NewStaticCredentials(cfg.CwAccessKey, cfg.CwSecretKey, "")
+		awsconf := aws.NewConfig().WithRegion(cfg.CwRegion).WithCredentials(cred)
+		hook, err := lc.NewBatchingHook(cfg.CwLogGroup, "rosocp", awsconf, 10*time.Second)
```

Comment:
@patilsuraj767 It would be great to have two log streams, one for API and another one for processor and recommender pods

Reply:
Will it be ok to have log stream name `rosocp-api` for API and `rosocp-backend` for processor and recommender?

Reply:
Yes, sounds good